starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1790,12 +1804,14 @@ sub print_report {
     if ( ( defined $self->{ra} ) and (defined $self->{dec}) and (defined $self->{roll})){
 	$o .= sprintf "RA, Dec, Roll (deg): %12.6f %12.6f %12.6f\n", $self->{ra}, $self->{dec}, $self->{roll};
     }
-    if ( defined $self->{DITHER_ON} && $self->{obsid} < $ER_MIN_OBSID ) {
-	$o .= sprintf "Dither: %-3s ",$self->{DITHER_ON};
+    if ( defined $self->{cmd_dither_status} ) {
+	$o .= sprintf "Dither: %-3s ",$self->{cmd_dither_status};
 	$o .= sprintf ("Y_amp=%4.1f  Z_amp=%4.1f  Y_period=%6.1f  Z_period=%6.1f",
-		       $self->{DITHER_Y_AMP}*3600., $self->{DITHER_Z_AMP}*3600.,
-		       360./$self->{DITHER_Y_FREQ}, 360./$self->{DITHER_Z_FREQ})
-	    if ($self->{DITHER_ON} eq 'ON' && $self->{DITHER_Y_FREQ} && $self->{DITHER_Z_FREQ});
+		       $self->{cmd_dither_y_amp}, $self->{cmd_dither_z_amp},
+		       $self->{cmd_dither_y_period}, $self->{cmd_dither_z_period})
+        if ($self->{cmd_dither_status} eq 'ENAB' && $self->{cmd_dither_y_period} && $self->{cmd_dither_z_period});
+	$o .= sprintf ("(no commanded dither amp: checked with 20 arcsec dither)")
```

Comment:
And this adds a new comment that mentions the amplitude used for the actual catalog check.  It just sticks it on the line previously reserved for the actual dither parameters.  This is also controversial.  And will probably need accompanying change in starcheck_parser code.

Reply:
Are we ready to use something for continuity for dither params?

Reply:
Kadi.commands is probably not ready for operational use like this.  It is close but we need to get daily checking in place and bring to LR.  So open a new issue for that.

Reply:
I'll have to see how the output looks with the new comment.